### PR TITLE
Add 'build' gulp task

### LINKED ISSUE #3: Add concat / uglify tasks

### PR DESCRIPTION
**Pull request by Dayde**
- Concat all sources
- Wrap them in an IIFE
- Uglify the result

Closes #3

### CODE REVIEW

File: wrap-template.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+(function(){
```
Comment (mjeanroy) [inline wrap-template.js]: You should add parameters to your anonymous function:

    (function(window, document, undefined) {
        // ....
    })(window, document, void 0);

File: gulpfile.js [hunk 2]
```diff
@@ -46,4 +49,14 @@ gulp.task('test', function(done) {
   }, done);
 });
 
-gulp.task('default', ['lint', 'test']);
+gulp.task('minify', function(done) {
+  return gulp.src(__dirname + '/src/**/*.js')
```
Comment (mjeanroy) [inline gulpfile.js]: File order is important here (look at karma configuration). I think you should probably put an array of file as parameter ?

File: gulpfile.js [hunk 3]
```diff
@@ -46,4 +49,14 @@ gulp.task('test', function(done) {
...
+    .pipe(concat('waffle.js'))
+    .pipe(wrap({src: __dirname + '/wrap-template.js'}))
```
Comment (mjeanroy) [inline gulpfile.js]: Is it possible to copy concat file to the build directory ?

File: wrap-template.js (hunk 1 above)
Comment (mjeanroy) [inline wrap-template.js]: You forgot licence header :)

Comment (Dayde) [inline wrap-template.js]: should licence header be kept after minification?

File: gulpfile.js (hunk 3 above)
Comment (Dayde) [inline gulpfile.js]: will do :)

**Comment by mjeanroy** [comment]
Two things:
- Please add your name to the copyright section when you change a file
- Please, do not remove license header of individual files :)

Anyway, good job :+1:

**Pull request merged**